helm/defectdojo/values.yaml

```diff
@@ -250,8 +250,14 @@ django:
     # could be emptyDir (not for production) or pvc
     type: emptyDir
     # in case if pvc specified, should point to the already existing pvc
-    persistentVolumeClaim: media
-
+    persistentVolumeClaim:
+      # set to true if django.mediaPersistentVolume.type is set to pvc
+      create: false
+      name:
+      size: 10Gi
```

Comment:
10Gi might a little to much as default? 1 oder 2 Gi should be enough. What do you think?

Reply:
I am not entirely sure how defectdojo uses storage though but I think we can have it set to 5Gi. How about that?

Reply:
It uses the storage only if users upload images or documents in defectdojo which is not the main usecase. 5Gi is fine for me, doesn't really matter.

Reply:
Ok cool, I will update that.